geopandas/geodataframe.py

```diff
@@ -398,6 +398,26 @@ def rename_geometry(self, col, inplace=False):
             self.rename(columns={geometry_col: col}, inplace=inplace)
             self.set_geometry(col, inplace=inplace)
 
+    @property
+    def active_geometry_name(self):
+        """Return a name of an active geometry column
```

Comment:
```suggestion
        """Return the name of the active geometry column
```

(given there is only one such column with one name)